umbraco/Umbraco-CMS
# V14: Public access controller

# Notes
- Implemented CRUD for Public access entries with the Attempt pattern
- For Create and Update, we're instead using a `PublicAccessEntrySlim` instead, as it removes a lot of bloat properties (derived from `IEntity`
- For `DeleteAsync` we're now just passing the key instead of the entire entity'

# How to test
- Create a doc type with allow as root
- Create 2 different content nodes.
- Call the 4 different endpoints (Create, Get, Update, Delete)
- If you have the old backoffice running, you should be able to see the page protection when you create it.
